toolchain/eslint-config/configs/svelte.js

```diff
@@ -48,6 +48,16 @@ export default defineFlatConfig([
 
       'svelte/block-lang': ['error', { script: 'ts', style: ['postcss', 'css'] }],
 
+      '@typescript-eslint/no-unused-vars': [
+        'error',
+        {
+          argsIgnorePattern: '^_',
+          varsIgnorePattern: '^_|\\${2}(Props|Slots|Events)',
```

Comment:
Need to override this rule to add the special $$ types used for typing components. These show as unused but because of the usage this is a false positive.